doc/source/api/core/solver/events.rst

```diff
@@ -4,16 +4,16 @@ EventsManager
 =============
 
 An instance of ``EventsManager`` exists as an active ``events_manager`` property in each
-solution-mode session object. It allows to register client callback with server side events.
-Whenever event occurs, the registered callback is triggered with session id and event info 
-passed as arguments to callback. It is useful in solution monitoring and updating graphics 
-during run time.
+solution-mode session object. It allows for registering the client callback with server-side
```

Comment:
It's not really *the* callback -  the original should have been pluralised. Also, we're not allowing _for_ something, we're simply allowing it. The meaning is different. However, the original grammar is incorrect. 

**You can register callbacks with the EventsManager, which are triggered by server-side events.**